### PR TITLE
[Clean up] Clean unused code

### PR DESCRIPTION
**Pull request by patrickvonplaten**
This PRs finishes the easy refactoring (just deleting old code).

All slow tests pass + stable diffusion pipeline was tested for subjective image quality and all looks good :+1: 

Now the big single letter variable renaming can begin

### CODE REVIEW

File: src/diffusers/modeling_utils.py [hunk 1]
```diff
@@ -390,7 +390,7 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
                 )
             except EntryNotFoundError:
                 raise EnvironmentError(
-                    f"{pretrained_model_name_or_path} does not appear to have a file named {model_file}."
+                    f"{pretrained_model_name_or_path} does not appear to have a file named {WEIGHTS_NAME}."
```
Comment (patrickvonplaten) [inline src/diffusers/modeling_utils.py:393]: `model` file is actually not yet defined when this error is hit

File: src/diffusers/models/attention.py [hunk 2]
```diff
@@ -1,12 +1,11 @@
 import math
-from inspect import isfunction
 
 import torch
 import torch.nn.functional as F
 from torch import nn
 
 
-class AttentionBlockNew(nn.Module):
+class AttentionBlock(nn.Module):
```
Comment (patrickvonplaten) [inline src/diffusers/models/attention.py:8]: Let's call it `AttentionBlock` and not `AttentionBlockNew`

**Comment by patrickvonplaten** [comment]
Solves https://github.com/huggingface/diffusers/issues/199

File: src/diffusers/models/attention.py [hunk 3]
```diff
@@ -234,7 +178,7 @@ def forward(self, x, context=None, mask=None):
         h = self.heads
 
         q = self.to_q(x)
-        context = default(context, x)
```
Comment (patrickvonplaten) [inline src/diffusers/models/attention.py:237]: death to one-line functions

File: src/diffusers/models/attention.py [hunk 4]
```diff
@@ -280,155 +224,3 @@ def __init__(self, dim_in, dim_out):
     def forward(self, x):
         x, gate = self.proj(x).chunk(2, dim=-1)
         return x * F.gelu(gate)
-
```
Comment (patrickvonplaten) [inline src/diffusers/models/attention.py:283]: All of this code is not used anymore

File: src/diffusers/models/unet_blocks.py [hunk 5]
```diff
@@ -17,8 +17,8 @@
 import torch
 from torch import nn
 
-from .attention import AttentionBlockNew, SpatialTransformer
-from .resnet import Downsample2D, FirDownsample2D, FirUpsample2D, ResnetBlock, Upsample2D
+from .attention import AttentionBlock, SpatialTransformer
+from .resnet import Downsample2D, FirDownsample2D, FirUpsample2D, ResnetBlock2D, Upsample2D
```
Comment (patrickvonplaten) [inline src/diffusers/models/unet_blocks.py:21]: 2D is more in line with all the other imports here

File: src/diffusers/models/attention.py [hunk 6]
```diff
@@ -234,7 +178,7 @@ def forward(self, x, context=None, mask=None):
         h = self.heads
 
         q = self.to_q(x)
-        context = default(context, x)
+        context = context if context is not None else x
```
Comment (patil-suraj) [inline src/diffusers/models/attention.py:181]: Much better :D

**Review by patil-suraj — APPROVED** [review]
Looks good! Thanks a lot for cleaning this.

**Pull request merged**